redirects.yaml

```diff
@@ -1029,281 +596,68 @@ support/faq/?: "/support"
 support/free/?: "https://community.ubuntu.com/"
 support/local: "https://community.ubuntu.com/t/local-communities/34"
 support/local/?: "https://community.ubuntu.com/t/local-communities/34"
-support/marketplace/?: "http://partners.ubuntu.com/find-a-partner"
-support/marketplace/asia/?: "http://partners.ubuntu.com/find-a-partner"
+support/marketplace/?: "https://partners.ubuntu.com/find-a-partner"
+support/marketplace/asia/?: "https://partners.ubuntu.com/find-a-partner"
```

Comment:
Couldn't the first one just be more aggressive?

Reply:
good call.